.circleci/create_circleci_config.py

```diff
@@ -397,6 +397,7 @@ def job_name(self):
 
 examples_torch_job = CircleCIJob(
     "examples_torch",
+    additional_env={"OMP_NUM_THREADS": 8},
```

Comment:
The original value `1` make some test slow and sometimes timeout (> 120 s)

Reply:
This feels like something which is going to make things very hard to debug and cause flaky tests: can we guarantee the same threads will receive the same jobs each time?

Reply:
This doesn't mean the jobs will be sent to different thread. It's really low level where python dispatches some tasks to different thread (even within a single process), I believe.

At the `test level`, that is controlled by `pytest -n x` , where is the number of works to run different tests. And so far we use `pytest -n 8` withtout having `same worker will receive the same jobs each time`.

Reply:
OK! :)